### PR TITLE
Add Marks Polakovs to students.

### PR DESCRIPTION
**Pull request by markspolakovs**
Part of GCI task https://codein.withgoogle.com/dashboard/task-instances/6285251135406080/

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://marksomnian.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -506,4 +506,11 @@
   blog: http://disam8853.github.io/
   twitter: Disam_Joker
   facebook: www.sam3
-  bio: I love everyone
\ No newline at end of file
+  bio: I love everyone
+
+- name: Marks Polakovs
+  github: marksomnian
+  image: markspolakovs.jpg
```
Comment (jigyasa-grover) [inline _data/students.yml]: _Error displaying picture_. Your image name is `MarksPolakovs.jpg` whereas you are trying to access a non-available `markspolakovs.jpg`

**Comment by jigyasa-grover** [comment]
@marksomnian Kindly rectify the mistake before we can merge.

**Comment by markspolakovs** [comment]
@jig08 Rectified, thanks for letting me know.

**Comment by jigyasa-grover** [comment]
@marksomnian Squash your commits. [This](https://github.com/fossasia/gci15.fossasia.org/blob/gh-pages/GIT.md) might help.

**Comment by markspolakovs** [comment]
@jig08 Squashed.

**Comment by mananwason** [comment]
@marksomnian Please resolve conflicts as well. Can't merge till the conflicts are resolved. Thanks!

**Comment by markspolakovs** [comment]
@jig08 Resolved.

**Comment by mananwason** [comment]
@marksomnian You need to remove the merge commit as well. Thanks!

**Comment by markspolakovs** [comment]
@mananwason Done.

**Comment by mananwason** [comment]
@marksomnian I believe you haven't added an image to the image folder. The link for your image is broken. Please fix that and squash the commits again.

**Comment by jigyasa-grover** [comment]
Okay @marksomnian , the picture shows but your PR shows 2 commits, kindly squash 'em.

**Comment by markspolakovs** [comment]
@jig08 Done, thanks for your patience.

**Comment by jigyasa-grover** [comment]
No issues @marksomnian  :) Your persistence is appreciated !

**Comment by mananwason** [comment]
Awesome @marksomnian !! Merging.

**Comment by markspolakovs** [comment]
@mananwason @jig08 Nice, thanks for persisting through all this Git nonsense.

**Pull request merged**